backend/authentication/models/user.py

```diff
@@ -24,7 +24,7 @@ def create_user(
             first_name,
             last_name,
             email,
-            password,
+            "",
```

Comment:
Isn't there a risk that someone authenticates with an empty password here?

Reply:
The auth token issuer is the SSO
Thus the portal credentials are no longer used
We will have to remove the login route on the portal side, but even if you use this route, the issued token will have the wrong signature